packages/server/observability/utils/requestContext.ts

```diff
@@ -1,11 +1,11 @@
-import { REQUEST_ID_HEADER } from '@/observability/components/express/expressLogging'
 import { asyncRequestContextEnabled } from '@/modules/shared/helpers/envHelper'
-import type express from 'express'
 import { AsyncLocalStorage } from 'node:async_hooks'
 import type { Logger } from 'pino'
 
 type StorageType = {
-  requestId: string
+  requestId?: string
```

Comment:
`{requestId: string} | {taskId: string, taskName}` would be better - one or the other must be set